.travis.yml

```diff
@@ -1,29 +1,23 @@
 language: c
 env:
+###############################################################################
   # FIXME: If we want to check assembly too SYNTAX_ONLY needs to be 0
   # FIXME: We'd like to build with WARNINGS_AS_ERRORS=1 but there are too
   # many warnings firing right now
   # Note: Can't use CC variable as TravisCI will override it
-  - C_COMPILER=gcc-4.8 SYNTAX_ONLY=1 WARNINGS_AS_ERRORS=0 VERBOSE=0 SUPPRESS_WARNINGS=1 REPORT_CC_FILE=2
+  - C_COMPILER=gcc SYNTAX_ONLY=1 WARNINGS_AS_ERRORS=0 VERBOSE=0 SUPPRESS_WARNINGS=1 REPORT_CC_FILE=2
```

Comment:
I wonder whether there is a better place for this definition? Currently running the checks locally is a little bit inconvenient, because one has to look at the `.travis.yml` file, understand what it does, copy the content of this line and run make manually with all these parameters. Maybe have two scripts `check-with-gcc.sh` and `check-with-clang.sh` that each contain the full call to make and can be called manually and at Travis?